playbooks/appsemblerPlaybooks/eucalyptus_enterprise_deploy.yml

```diff
@@ -0,0 +1,106 @@
+---
+
+# Appsembler pro tier deployment
+# Docs: https://github.com/appsembler/openedx-docs/blob/master/openedx/pro_tier_deployment.md
+
+- name: Install Mongo Replica Set
+  hosts: "mongo-servers"
+  sudo: True
+  gather_facts: True
+  vars:
+    BACKUPS_MONGO: True
+    BACKUPS_MYSQL: False
+    COMMON_ENABLE_BACKUPS: True
+  roles:
+    - { role: swapfile, SWAPFILE_SIZE: "2GB" }
+    - mongo_3_0
+    - backups
+
+
+- name: Install Mongo Backups
+  hosts: "mongo-arbiter"
```

Comment:
From what I gathered the last time i tried to get clustered mongo running is that there is not notion of a mongo arbiter in the upstream mongo_3_0 or mongo roles. We should not be using this. Since we have 3 nodes anyway I think it's better to make all 3 data nodes and not worry about an arbiter since we have a odd server number anyway. cc @mrgnr